src/server/src/entities/verificationKey.ts

```diff
@@ -0,0 +1,25 @@
+import {
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    JoinColumn,
+    OneToOne,
+    BaseEntity,
+} from 'typeorm';
+import { User } from './user';
+
+@Entity()
+export class VerificationKey extends BaseEntity {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column()
+    userId: number;
+
+    @OneToOne(() => User)
+    @JoinColumn()
+    user: User;
+
+    @Column()
+    verificationKey: string;
```

Comment:
Kind of redundant for this column to have the same name as the table. Maybe just use "key"?

Reply:
Done.